py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/convert.py

```diff
@@ -69,7 +74,11 @@ def convert_files(filepaths: list[Path], config: Configuration) -> pa.Table:
         else:
             failed_ingestion.append(ret_obj)
 
-    pool.close()
-    pool.join()
+    logging.info(
+        "Completed converting %d files with config %s" % (len(filepaths),
+                                                          config.config_type))
+
+    if len(failed_ingestion) > 0:
+        logging.warning("Unable to process %d files" % len(failed_ingestion))
```

Comment:
should figure out a better way to handle this other than just logging?

Reply:
I think this will depend on how we utilize the logging that is being sent to splunk and how frequently (if ever) this occurs.